modules/apps/foundation/frontend-js/frontend-js-loader-modules-extender/src/main/java/com/liferay/frontend/js/loader/modules/extender/npm/model/JSModuleAdapter.java

```diff
@@ -23,10 +23,24 @@
 import java.util.List;
 
 /**
+ * A basic incomplete implementation of {@link JSModule} that can be reused by
+ * real implementations.
  * @author Iván Zaera
  */
 public abstract class JSModuleAdapter implements JSModule {
 
+	/**
+	 *
+	 * @param jsPackage the package where this module belongs
+	 * @param name the name of the module
+	 * @param url the canonical URL of this module
+	 * @param resolvedURL the URL that gives access to the module to which this
+	 *                    module is resolved (in the end, that can be this same
+	 *                    module or another copy of this module contained in a
+	 *                    different bundle)
```

Comment:
@izaera Do you think it's appropriate to add this info to the `JSModuleAdapter` class description? Or somewhere else?